lib/code0/identities/provider/oidc.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Code0
+  module Identities
+    module Provider
+      class Oidc < BaseOauth
+        def token_url
+          config[:token_url]
+        end
+
+        def token_payload(code)
+          { code: code,
+            grant_type: "authorization_code",
+            redirect_uri: config[:redirect_uri],
+            client_id: config[:client_id],
+            client_secret: config[:client_secret] }
+        end
+
+        def user_details_url
+          config[:user_details_url]
+        end
+
+        def authorization_url
+          config[:user_details_url]
```

Comment:
At least we need to change this
```suggestion
          config[:authorization_url]
```

But I would probably go to something like this:
```suggestion
          config[:authorization_url].gsub("{client_id}", config[:client_id]).gsub("{redirect_uri}", config[:redirect_uri])
```

Reply:
I accepted your suggestion